docs/source/admin/traffic_vault.rst

```diff
@@ -79,6 +79,53 @@ Administration of the PostgreSQL database for Traffic Vault
 
 Similar to administering the Traffic Ops database, the :ref:`admin <database-management>` tool should be used for administering the PostgreSQL Traffic Vault backend.
 
+.. program:: reencrypt
+
+app/db/reencrypt/reencrypt
+--------------------------
+The :program:`reencrypt` binary is used to re-encrypt all data in the Postgres Traffic Vault with a new base64-encoded AES key.
+
+.. note:: For proper resolution of configuration files, it's recommended that this binary be run from the ``app/db/reencrypt`` directory.
+
+Usage
+"""""
+``./reencrypt [options]``
+
+Options and Arguments
+"""""""""""""""""""""
+.. option:: --new-key NEW_KEY
+
+	(Optional) The file path for the new base64-encoded AES key. Default is ``/opt/traffic_ops/app/conf/new.key``.
+
+.. option:: --previous-key PREVIOUS_KEY
+
+	(Optional) The file path for the previous base64-encoded AES key. Default is ``/opt/traffic_ops/app/conf/aes.key``.
+
+.. option:: --cfg CONFIG_FILE
+
+	(Optional) The path for the configuration file. Default is ``./reencrypt.conf``.
+
+.. option:: --help
+
+	(Optional) Print usage information and exit.
+
+.. code-block:: bash
+	:caption: Example Usage
+
+	./reencrypt --new-key ~/exampleNewKey.txt --previous-key ~/exampleOldKey.txt
+
+reencrypt.conf
+""""""""""""""
+This file deals with configuration of the Traffic Vault Database to be used with the :program:`reencrypt` tool.
+
+:dbname: The name of the PostgreSQL database used.
+:hostname: The hostname (:abbr:`FQDN (Fully Qualified Domain Name)`) of the server that runs the Traffic Vault Database.
+:password: The password to use when authenticating with the Traffic Vault database.
+:port: The port number (as a string) on which the Traffic Vault Database is listening for incoming connections (NOTE: the PostgreSQL default is 5432).
```

Comment:
My bad, maybe this wasn't a total copy/paste of database.conf. This re-encrypt tool expects a string, not an int. For convenience/consistency, should we just make it parse as a string instead of an int? Then we can just reuse database.conf to run this tool.

Reply:
Oh, nevermind, we won't be able to reuse database.conf since that's TODB not TVDB. It should parse as an int, so this line should not have `(as a string)`.

Reply:
oh dang it. ill get that note out of there. i think we should leave it as a separate conf file. we couldnt use database.conf (since thats for TO DB) but I thought about using the traffic vault info from cdn.conf and decided it would be better as a completely stand alone tool. this way we could reencrpyt a different db than is in use (like for testing or something) without impacting TO at all

Reply:
i should have refreshed the page before commenting back haha. we're on the same page